Lil-Peep/src/styles/modules/site-header.scss

```diff
@@ -1,9 +1,238 @@
+        text-decoration: none;
+      }
+
+      .site-logo {
+        left: 90%;
+        position: relative;
+      }
+
+      .site-header__logo-image {
+        left: 50%;
+        position: relative;
+      }
+      .searchlogin {
+        align-items: center;
+        display: flex;
+        justify-content: flex-end;
+        width: 50%;
+      }
+      .customer-login {
+        li {
+          display: flex;
+          margin-right: 20px;
+        }
+        ul {
+          display: flex;
+          margin-right: 20px;
+        }
+        .site-logo {
+          display: flex;
+
+          img {
+            display: block;
+            justify-content: center;
+            width: 100%;
+          }
+        }
+      }
+    }
+  }
+
+  .site-nav {
+    display: flex;
+    justify-content: space-around;
+    list-style: none;
+
+    a {
+      color: #627387;
+      text-decoration: none;
+    }
+
+    a:hover {
+      color: red;
+      text-decoration: underline;
+    }
+  }
+}
+
+
+@media only screen and ( max-width: 640px) {
+
+  .section-header{
+    display: none;
+  }
+
+  .header-mobile-section {
+    align-items: baseline;
+    display: flex;
+    justify-content: space-between;
+
+    .iconmenu-left {
+      background-color: white;
+      border: none;
+    }
+
+    .header-logo {
+      margin-bottom: 0px;
+    }
+  }
+
+  .iconmenu1 {
+    align-items: center;
+    background: transparent;
+    border: none;
+    color: #627387;
+    display: flex;
+    font-size: 16px;
+  }
+
+  .sidenav {
+    background-color: white;
+    height: 100%;
+    left: 0;
+    overflow-x: hidden;
+    position: fixed;
+    top: 0;
+    transition: 0.5s;
+    width: 0;
+    z-index: 1;
+
+    .sidenav-logo {
+      display: flex;
+      height: 50px;
+      justify-content: space-around;
+      margin-bottom: 10px;
+
+    .closebtn {
+      background-color: white;
+      border: none;
+      color: #627387;
+      padding-top: 20px;
+      }
+    }
+  }
+
+  .header-mobile {
+    background-color: #627387;
+    padding-bottom: 5px;
+    padding-top: 5px;
+
+      li{
+        list-style: none;
+      }
+
+      a{
+        color: #FFFFFF;
+        display: block;
+        font-size: 16px;
+        list-style: none;
+        padding: 8px 8px 8px 32px;
+        text-decoration: none;
+        transition: 0.3s;
+      }
+      a:hover {
+        text-decoration: underline;
+      }
+      ul {
+        padding-left: 20px;
+      }
+      .line {
+        border: 2px solid #8994A1;
+        box-sizing: border-box;
+        height: 2px;
+        width: 228px;
+      }
+  }
+
+  .footer-mobile{
+    color: #72808C;
+    text-align: center;
+
+    .social-media {
+      margin-top: 25px;
+    }
+
+    .social-sharing {
+      align-content: center;
+      display: flex;
+      justify-content: center;
+    }
+
+    .smi {
+      align-items: center;
+      background-repeat: no-repeat;
+      border-radius: 3px;
+      border: 2px solid #72808C;
```

Comment:
Suggestion - You should use variables for colors... for example.
$name-color: #72808C;
border: 2px solid $name-color;